embedded-graphics/src/fonts/font24x32.rs

```diff
@@ -0,0 +1,243 @@
+use crate::fonts::font_builder::{FontBuilder, FontBuilderConf};
+
+#[derive(Debug, Copy, Clone)]
+pub enum Font24x32Conf {}
+impl FontBuilderConf for Font24x32Conf {
+    const FONT_IMAGE: &'static [u8] = include_bytes!("../../data/font24x32_1bpp.raw");
+    const CHAR_HEIGHT: u32 = 32;
+    const CHAR_WIDTH: u32 = 24;
+    const FONT_IMAGE_WIDTH: u32 = 960;
+    fn char_offset(c: char) -> u32 {
+        let fallback = '?' as u32 - ' ' as u32;
+        if c < ' ' {
+            return fallback;
+        }
+        if c <= '~' {
+            return c as u32 - ' ' as u32;
+        }
+        fallback
+    }
+}
+
+/// 24x32 pixel monospace font
+///
+/// There is also the [`text_24x32`] macro to provide an easier to use interface.
+///
+/// The 24x32 font is just a scaled up version of the 12x16 font, so [![here is the 12x16 font spritemap screenshot](https://raw.githubusercontent.com/jamwaffles/embedded-graphics/master/embedded-graphics/data/font12x16.png)](https://raw.githubusercontent.com/jamwaffles/embedded-graphics/master/embedded-graphics/data/font12x16.png)
+///
+/// # Examples
+///
+/// See the [module-level documentation](./index.html) for examples.
+///
+/// [`text_24x32`]: ../macro.text_24x32.html
+pub type Font24x32<'a, C> = FontBuilder<'a, C, Font24x32Conf>;
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use crate::fonts::Font;
+    use crate::geometry::{Dimensions, Point, Size};
+    use crate::mock_display::MockDisplay;
+    use crate::pixelcolor::BinaryColor;
+    use crate::style::WithStyle;
+    use crate::transform::Transform;
+    use crate::Drawing;
+
+    const WIDTH: usize = Font24x32Conf::CHAR_WIDTH as usize;
+    const HEIGHT: usize = Font24x32Conf::CHAR_HEIGHT as usize;
+    const HELLO_WORLD: &'static str = "Hello World!";
+
+    #[test]
+    fn off_screen_text_does_not_infinite_loop() {
```

Comment:
I _think_ this test was made redundant by #158 - negative coordinate checks were moved out of e-g into the drivers' domain of responsibility. Seeing as you're refactoring the font tests anyway, can you remove all these `off_screen_text_does_not_infinite_loop` tests?

If you think it should be left in, the Y value in the `.translate()` call should be larger than the font height to shift the entire font into negative coordinates.